scm-ui/ui-components/src/forms/FileInput.tsx

```diff
@@ -49,9 +51,16 @@ const FileInput: FC<Props> = ({
   className,
   ref,
   onBlur,
-  onChange
+  onChange,
 }) => {
+  const [t] = useTranslation("commons");
+  const [file, setFile] = useState<File | null>(null);
+
   const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const uploadedFile = event?.target?.files![0];
+    // @ts-ignore the uploaded file doesn't match our types
+    setFile(uploadedFile);
```

Comment:
You copied this from the FileUpload component. Do we really need this? The filename was also shown before correctly. I think we should look for a better solution. You may ask @Pilopa about this.

Reply:
Good question. Due to the custom styling, the input itself is no longer visible and only a span.file-name is available in which the file name must be set somehow...

Reply:
@eheimbuch Its okay like this. There is unfortunately no better solution that I am aware of after spending some time researching the issue.